content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md

```diff
@@ -316,8 +316,9 @@ without compromising the minimum required capacity for running your workloads.
     {{< tabs name="k8s_kubelet_and_kubectl" >}}
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
     # replace x in 1.14.x-00 with the latest patch version
-    apt-get update
-    apt-get install -y kubelet=1.14.x-00 kubectl=1.14.x-00
+    apt-mark unhold kubelet kubectl && \
+    apt-get update && apt-get install -y kubelet=1.14.x-00 kubectl=1.14.x-00 && \
+    apt-mark hold kubelet kubectl
```

Comment:
would you care updating the debian/ubuntu tabs on older upgrade docs too?

Reply:
I thought that I would fix it together if there was an error.
However, there was no mistake in the old document itself, but should I format the old document according to the present form?

Reply:
ok, no need to change the old docs in that case.